bindings/http/http.go

```diff
@@ -38,10 +38,12 @@ import (
 )
 
 const (
-	MTLSEnable     = "MTLSEnable"
-	MTLSRootCA     = "MTLSRootCA"
-	MTLSClientCert = "MTLSClientCert"
-	MTLSClientKey  = "MTLSClientKey"
+	MTLSEnable        = "MTLSEnable"
+	MTLSRootCA        = "MTLSRootCA"
+	MTLSClientCert    = "MTLSClientCert"
+	MTLSClientKey     = "MTLSClientKey"
+	traceparentHeader = "traceparent"
```

Comment:
```suggestion
	TraceparentHeaderKey = "traceparent"
```

Constants should be named as such and should always be capitalized.